tools/reviewer/job/sync/GithubSync.java

```diff
@@ -56,53 +44,263 @@
   private static Flag<String> reviewerDiffLink =
       Flag.create("https://startupos-5f279.firebaseapp.com/diff/");
 
+  @FlagDesc(name = "diffs_to_sync", description = "List of Reviewer Diff number to sync")
+  private static final Flag<List<Long>> diffsToSync =
+      Flag.createLongsListFlag(Collections.emptyList());
+
   public static void main(String[] args) throws IOException {
     Flags.parseCurrentPackage(args);
-    GithubSync githubSync = new GithubSync();
     GithubClient githubClient = new GithubClient(login.get(), password.get());
 
-    if (args.length != 0) {
-      switch (args[0]) {
-        case "read":
-          {
-            githubSync.readPullRequest(githubClient);
-            break;
+    // Go over all Reviewer Diff numbers to sync
+    for (long diffNumber : diffsToSync.get()) {
+      syncWithGithub(githubClient, diffNumber);
+    }
+  }
+
+  private static void syncWithGithub(GithubClient githubClient, long diffNumber)
```

Comment:
Generally when a method becomes very long - we have 2 main strategies to cope with it:
* Extract repetitive parts to another method, sometimes with a parameter that changes the code to fit the different cases.
* Extract non-repetitive parts to another method, usually breaking the code into parts. For example, one part could be left in the original method, and the other part in the extracted method.

There are refactoring tools in IDEs that can help with this.

We should aim to make methods no longer than 40 lines, but it's not a strict rule. Sometimes it's ok if they're longer.